etl/steps/data/grapher/wb/2024-12-17/bilateral_remittance.py

```diff
@@ -0,0 +1,68 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+indicator_title = "Estimated bilateral remittance flows"
+description_short = "The estimated bilateral remittance flows between countries. These estimates use migrant stocks, host country incomes, and origin country incomes."
```

Comment:
I think aside from indicator_title you can remove these as they persist from garden step